scripts/e2e-ci.sh

```diff
@@ -145,7 +145,7 @@ test_init() {
 
     elif [ "$LARGE" -eq 1 ]; then
         step Initalizing ilab with l40s-x4 train profile
-        ilab config init --non-interactive --train-profile="${SCRIPTDIR}/test-data/train-profile-l40s-x4.yaml"
+        ilab config init --non-interactive --profile="${SCRIPTDIR}/test-data/train-profile-l40s-x4.yaml"
```

Comment:
Can we change the name of the file from `train-profile-l40s-x4.yaml` to `profile-l40x-x4.yaml`.

@nathan-weinberg or I can do the follow up work needed to fill out the rest of that file.

Reply:
sure! I can do that. 

I might do that work as part of this to account for testing

Reply:
took care of this